routes.yaml

```diff
@@ -0,0 +1,5 @@
+app_name: limited-partnerships-api
```

Comment:
we don't need this, we're running on ECS now

Reply:
Removed this line.